example-apps/html-js/index.html

```
@@ -0,0 +1,121 @@
+  </nav>
+
+  <main class="container margin-vert--lg">
+    <h1>Img2Num html-js Example Application</h1>
+    <p>A simple demo app to show how to use <a href="https://img2num.dev/docs/js/">Img2Num's JavaScript library</a>.</p>
+    <p>Upload an image to convert it to an SVG</p>
+
+    <input type="file" id="fileInput" accept="image/*" />
+
+    <section class="margin-top--md">
+      <h4>Original</h4>
+      <img id="original" />
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)